tests/pipeline/test_alias.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+
+from zipline.testing.predicates import assert_equal
+from zipline.pipeline import Classifier, Factor, Filter
+from zipline.utils.numpy_utils import float64_dtype, int64_dtype
+
+from .base import BasePipelineTestCase
+
+
+class WithAlias(object):
```

Comment:
I would probably inherit from `BasePipelineTestCase` in `WithAlias`, since this is where we're actually defining the functionality that depends on the superclass.